l10n_fr_naf_ape/partner.py

```diff
@@ -0,0 +1,34 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2011 Numérigraphe SARL.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.osv import osv, fields
+
+
+class Partner(osv.Model):
+    """Add the French APE (official main activity of the company)"""
+    _inherit = 'res.partner'
+    _columns = {
+        'ape_id': fields.many2one(
+            'res.partner.category', 'APE',
+            help="If the partner is a French company, enter it's official "
+                 "main activity in this field. The APE is chosen among the "
+                 "NAF nomenclature."),
+    }
```

Comment:
question: I'm wondering if this ape_id should be added to the fields returned by res_partner._commercial_fields() to be copied to contacts (cf. https://github.com/odoo/odoo/blob/8.0/openerp/addons/base/res/res_partner.py#L446)